### ISSUE TITLE
Swipe gesture to go back to conversations list does not work when opening Signal from a notification

### ISSUE THREAD

**Issue by Doge7734** [signalapp/Signal-iOS#5292]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
When tapping on a notification from Signal that opens up to a chat, it is not possible to swipe from the left edge of the screen to the right to go to the conversations list. Instead it to replies to a message. Tapping the back arrow and then going back into the conversation fixes the issue. The issue does not occur when manually opening the app.

### Steps to reproduce
- Tap on notification
- Try swiping left to right from the left edge of the screen

**Actual result:** Cannot return to conversation lists. Replies to a chat instead.

**Expected result:** Return to conversations lists.

### Device info

**Device**: iPhone 12 Mini

**iOS version**: 15.3.1

**Signal version:** 5.31.0.14

### Link to debug log
 https://debuglogs.org/ios/5.31.0/85b66fbe8cc4dd94a50fc16904a106079ef65ce9b22d97f90a7e595bd348bf34.zip

**Comment by TheMuntan** [comment]
I never realized why it is that I can sometimes swipe from the left to go back but other times I wasn't able to. But this post made me realize that it is exactly what you described!

**Comment by Doge7734** [comment]
Good to know that someone else is having that problem. I actually forgot I had made this bug report. I was wondering if it had been fixed since I now have a de-Googled CalyxOS device (though I am still keeping my iPhone in case I want to go back) and can't check. I'm guessing it hasn't yet. It probably isn't high on the list as it is only a minor bug.

**Comment by azuredusk10** [comment]
I don’t think this is to do with opening messages from a notification, but rather, which orientations you locked and unlocked your phone in. I can reproduce this issue in another way in v 6.6.0 (10). 

1. Open a chat in Signal
2. Turn phone to landscape 
3. Lock the phone
4. While locked, rotate the phone to portrait 
5. Unlock the phone
6. Try swiping from the left edge of the screen. 

Expected behaviour: navigate back to the conversation list.

Actual behaviour: The gesture no longer works, and instead it always quotes a message. 

I think this is because the “swipe to go back” gesture is only enabled when the phone is in portrait. When locking the phone while in landscape, rotating the phone to portrait, then unlocking the phone, I don’t think Signal is correctly re-enabling the “swipe to go back” gesture

**Comment by Doge7734** [comment]
I tried your steps but I am still able to swipe to go back to the conversations list. It only happens if I open Signal via a notification.
